src/core/Akka.Tests/IO/SimpleDnsCacheSpec.cs

```diff
@@ -79,5 +80,96 @@ public void Cache_should_be_updated_with_the_latest_resolved()
             cache.Put(cacheEntryTwo, ttl);
             cache.Cached("test.local").ShouldBe(cacheEntryTwo);
         }
+
```

Comment:
This test has used the old version with a value type, therefore I have copied the old version into this class.